src/api/java/com/minecolonies/api/configuration/ServerConfiguration.java

```diff
@@ -29,7 +29,6 @@
     public final ForgeConfigSpec.IntValue     initialCitizenAmount;
     public final ForgeConfigSpec.BooleanValue builderPlaceConstructionTape;
     public final ForgeConfigSpec.BooleanValue allowInfiniteSupplyChests;
-    public final ForgeConfigSpec.BooleanValue allowInfiniteColonies;
```

Comment:
why's this getting removed?

Reply:
All it does (as far as I could tell) was change the message shown when placing a new town hall

Reply:
Yes, it allows to abandon colonies instead of only allowing to delete them. We should keep this

Reply:
thx, I'll add it back
could you send a link to where it does that (so I know how to find it for next time)? I searched and the config, lang file, and WindowTownHallColonyManage.java were the only places I found it

Reply:
@Raycoms do you have a link or sth?

Reply:
Double checked with someaddons, it's sth that's currently missing in the UI. So if you want you can add that to the UI. Should be pretty straight forward. You can ask us for tips if you need

Reply:
Add it back for this PR, and then in another PR you can adjust the UI to respond to this.

Reply:
Okay, will do